src/twisted/internet/interfaces.py

```diff
@@ -1340,7 +1347,7 @@ class IReactorFromThreads(Interface):
     @since: 15.4
     """
 
-    def callFromThread(callable, *args, **kw):
+    def callFromThread(f: Callable[..., Any], *args, **kw):
```

Comment:
This is still an interface change callable -> f

Reply:
I took this out